### ISSUE TITLE
New-UDEndpoint : allow passing query string parameters

### ISSUE THREAD

**Issue by apetitjean** [ironmansoftware/universal-dashboard#67]
Please implement a way of passing data like this:
 GET /users?location=US&age=21

Because today everything after "?" is ignored.

**Comment by adamdriscoll** [comment]
This has been implemented in 1.6.0-beta1

**Comment by apetitjean** [comment]
My bad... it works! Even in 1.5.3!

In fact I didn't know everything was in the $Request "automatic" variable. 
I used to the the content of my param (ie. $value) and not $Request

Here a chunk of code that works:
```PowerShell
$Endpoint0 = New-UDEndpoint -Url "/test/:value" -Method "GET" -Endpoint {
    param($value)

    $request | ConvertTo-Json
} 
```
